app/src/main/res/layout/app_widget_classic.xml

```diff
@@ -3,17 +3,19 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/content"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="match_parent"
     android:background="@drawable/card"
     android:orientation="horizontal"
     tools:ignore="ContentDescription">
 
     <ImageView
         android:id="@+id/image"
-        android:layout_width="@dimen/app_widget_classic_image_size"
-        android:layout_height="@dimen/app_widget_classic_image_size"
-        android:background="@drawable/default_album_art"
-        android:scaleType="centerCrop" />
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:scaleType="centerInside"
```

Comment:
There are two options for that:
The first image is with the value set to "centerCrop".
The second one with "centerInside".
<img src="https://user-images.githubusercontent.com/26522190/28571956-67c76fee-7145-11e7-91f5-0846d55ee8c9.png" width="500" />

In this way the albums image will be responsive and won't be cropped.